tools/check-templates

```diff
@@ -44,10 +50,19 @@ def check_html_templates(templates, modified_only):
     # there are also cases where Casper deliberately uses invalid HTML,
     # so we exclude them from our linter.
     templates = filter(
-        lambda fn: ('base.html' not in fn) and ('casperjs' not in fn),
+        lambda fn: (('base.html' not in fn) and ('casperjs' not in fn)),
         templates)
     templates = sorted(list(templates))
 
+    master_id_record = check_for_duplicate_ids(templates)
+
+    for ids in master_id_record:
```

Comment:
I doubt if this would work since `master_id_record.values()` would return a list of lists containing strings. But from what i understand from this, something like this should workout 
`for ids, fns in iteritems(template_id_dict):
        if len(fns) > 1:
            # print(...)
            for fn in fns:
                # ...
`
`iteritems()` is imported from `six` which maintains python 2 and 3 compatibility.